include/rlcg.hpp

```diff
@@ -71,6 +71,8 @@ class ReversibleLCG {
     uint64_t x;
 public:
     ReversibleLCG(unsigned int seed) : x(seed){}
+    // Hi 32-bits is the initial value, lo 32-bit is a random seed
+    void seed(uint64_t s) { x = s; }
```

Comment:
I think the name `seed` is a little bit confusing. People might be tempted to seed with time and so get unexpectedly (from their perspective) close initial values.

How about naming it something like `set_state` instead?

Reply:
> I think the name `seed` is a little bit confusing. People might be tempted to seed with time and so get unexpectedly (from their perspective) close initial values.
> 
> How about naming it something like `set_state` instead?

Sure, I made the change.